packages/mobile/src/page/Cafeteria/CafateriaBody/index.tsx

```diff
@@ -18,31 +17,41 @@ type Props = {
 function CafeteriaBody({ fullDate, day, selectedMenu }: Props) {
   const isHoliday = day === 6 || day === 0;
   const { data: cafeteriaMenu } = useCafeteriaQuery(selectedMenu, fullDate);
+  const isCafeteriaExist = cafeteriaMenu && cafeteriaMenu.length > 0;
 
   return (
     <main
       className={classnames($.cafeteria, {
-        [$["no-menu"]]: cafeteriaMenu && !cafeteriaMenu.length,
+        [$["no-menu"]]: !isCafeteriaExist,
       })}
     >
-      {cafeteriaMenu &&
-        cafeteriaMenu.length > 0 &&
-        cafeteriaMenu.map(({ menu, time }) => {
-          const [ mealTime, timeInfo ] = getCafeteriaTime(
-            isHoliday,
-            selectedMenu,
-            time,
-          );
-          return (
-            <CafeteriaMenuCard
-              key={menu}
-              {...{ mealTime, timeInfo }}
-              mealMenu={menu}
-            />
-          );
-        })}
+      {isCafeteriaExist && (
+        <>
+          {cafeteriaMenu.map(({ menu, time }) => {
+            const [ mealTime, timeInfo ] = getCafeteriaTime(
+              isHoliday,
+              selectedMenu,
+              time,
+            );
+            return (
+              <CafeteriaMenuCard
+                key={menu}
+                {...{ mealTime, timeInfo }}
+                mealMenu={menu}
+              />
+            );
+          })}
+          <ShareButton
+            size={12}
+            stroke="#9FB0C6"
+            successMsg="식단 링크가 클립보드에 복사되었습니다."
+            className={$["share-button"]}
+            text={<span className={$["share-button-text"]}>식단 공유하기</span>}
+          />
```

Comment:
```suggestion
          <ShareButton
            size={12}
            stroke="#9FB0C6"
            successMsg="식단 링크가 클립보드에 복사되었습니다."
            className={$["share-button"]}
          >
            <span className={$["share-button-text"]}>식단 공유하기</span>
          </ShareButton>
```
이런식으로 써도 좋을 것 같아요~ DefaultProps에 children이 정의되어있고 
기존코드에서 단일의 element를 넘길때는 children으로 보냈거든요!